README.md

```diff
@@ -1,43 +1,56 @@
 # CellProfiler-RunOnCluster 
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3275888.svg)](https://doi.org/10.5281/zenodo.3275888)
 
-A CellProfiler module for submitting batch jobs to a cluster running slurm.
+A CellProfiler module for submitting batch jobs to a cluster running Slurm, tested with [CellProfiler 4.2.1 and CellProfiler 4.2.2rc](https://cellprofiler.org/releases) on Windows 10 and 11.
```

Comment:
The new detail is not the first key message we want to deliver to users; this should be moved further down the README (or at least out of the first sentence).